sgx_ci/README.md

```diff
@@ -0,0 +1,130 @@
+# KMS inside an enclave
```

Comment:
This README is a copy right? Can we remove it or just remove & adapt the content for the specific case of the CI?